lasagne/__init__.py

```diff
@@ -28,7 +28,11 @@
 from . import updates
 from . import utils
 
-
 import pkg_resources
-__version__ = pkg_resources.get_distribution("Lasagne").version
+
+try:
+    __version__ = pkg_resources.get_distribution("Lasagne").version
+except pkg_resources.DistributionNotFound:  # pragma: no cover
+    pass
```

Comment:
Maybe set `__version__ = 'n/a'` in that case?  So code that checks for `__version__` doesn't fail.